Microsoft.Alm.Authentication/Secret.cs

```diff
@@ -42,6 +42,18 @@ public static string UriToName(TargetUri targetUri, string @namespace)
             return targetName;
         }
 
+        public static string UriToUrl(TargetUri targetUri, string @namespace)
+        {
+            BaseSecureStore.ValidateTargetUri(targetUri);
+            if (String.IsNullOrWhiteSpace(@namespace))
+                throw new ArgumentNullException(@namespace);
+
+            string targetName = $"{@namespace}:{targetUri.ToString(false, true, true)}";
```

Comment:
Ahha - that makes sense... I think. BitBucket doesn't care about the path at all?